.travis/script.sh

```diff
@@ -626,6 +626,12 @@ build_samr21() {
     REFERENCE_DEVICE=1 COVERAGE=1 PYTHONUNBUFFERED=1 OT_NCP_PATH="$(pwd)/$(ls output/posix/*/bin/ot-ncp)" RADIO_DEVICE="$(pwd)/$(ls output/*/bin/ot-rcp)" NODE_TYPE=ncp-sim make -f src/posix/Makefile-posix check || die
 }
 
+[ $BUILD_TARGET != posix-app-spi ] || {
+    ./bootstrap || die
+    # enable code coverage for OpenThread posix radio
+    REFERENCE_DEVICE=1 COVERAGE=1 READLINE=readline RCP_SPI=1 make -f src/posix/Makefile-posix || die
```

Comment:
How about add a command line to run the output, although it fails, it can still cover the failure case, and increase the code coverage.
```suggestion
    REFERENCE_DEVICE=1 READLINE=readline RCP_SPI=1 make -f src/posix/Makefile-posix || die
```